File: app/src/main/java/com/example/bullsandcows/DevActivity.java
package com.example.bullsandcows;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

public class DevActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_dev);
        findViewById(R.id.home_button).setOnClickListener(v-> {
            Intent intent = new Intent(this, MainActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/example/bullsandcows/DevActivity.kt
package com.example.bullsandcows

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class DevActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_dev)
        findViewById<Button>(R.id.home_button).setOnClickListener {
            val intent = Intent(this@DevActivity, MainActivity::class.java)
            startActivity(intent)
    }
    }
}